cmake/LLVMIRUtil.cmake

```diff
@@ -111,9 +111,25 @@ function(llvmir_attach_bc_target)
   # compile lang flags
   llvmir_extract_lang_flags(IN_LANG_FLAGS ${LINKER_LANGUAGE})
 
+  # Find all header files
+  foreach(IN_FILE ${IN_FILES})
+    get_filename_component(FILE_EXT ${IN_FILE} LAST_EXT)
+    get_filename_component(FILE_DIR ${IN_FILE} DIRECTORY)
+    if ("${FILE_EXT}" STREQUAL ".h" OR "${FILE_EXT}" STREQUAL ".hpp")
```

Comment:
This needs to be a bit more rigorous; maybe make `FILE_EXT` lowercase and compare it against a list (i.e., `.h`, `.hpp`, `.hxx`

Reply:
Done. Thanks!